Display.lua

```diff
@@ -649,11 +650,28 @@ function WarpDeplete:UpdateObjectivesDisplay()
     if boss.time ~= nil then
       if boss.time > 0 then
         local completionTimeStr = Util.formatTime(boss.time)
+        local bestDiffStr = ""
+
+        if timingsDisplayStyle ~= "hidden" then
+          local diff = nil
+          if timingsDisplayStyle == "bestDiff" then
+            local bestTime = self:GetBestTime(i)
+            if bestTime ~= nil then diff = boss.time - bestTime end
+          elseif timingsDisplayStyle == "lastDiff" then
+            local lastTime = self:GetLastTime(i)
+            if lastTime ~= nil then diff = boss.time - lastTime end
+          end
+
+          if diff ~= nil then
+            local color = diff <= 0 and self.db.profile.timingsImprovedTimeColor or self.db.profile.timingsWorseTimeColor
+            bestDiffStr = "[|c" .. color .. Util.formatTime(diff, true) .. "|r|c" .. completionColor .. "]"
+          end
+        end
 
         if alignStart then
-          objectiveStr = "[" .. completionTimeStr .. "] " .. objectiveStr
+          objectiveStr = bestDiffStr .. "[" .. completionTimeStr .. "] " .. objectiveStr
         else
-          objectiveStr =  objectiveStr .. " [" .. completionTimeStr .. "]"
+          objectiveStr =  objectiveStr .. " [" .. completionTimeStr .. "]" .. bestDiffStr
```

Comment:
Since we're adding more and more options here, I think the best case for me would be to have a few presets but allow a custom format string, just like for the forces bar.

I don't necessarily see this as a requirement for this PR though, but you could insert a `TODO` here for that.

Reply:
I've added a TODO comment. I'm currently very busy, so implementing that with this PR would delay it even more. I've opened Issue #50 for this feature, so it doesn't get lost.